live-build/variants/internal-dct/ansible/playbook.yml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2024 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- hosts: all
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  roles:
+    - appliance-build.minimal-common
+    - appliance-build.minimal-internal
+    - appliance-build.dct-common
+    - appliance-build.masking-common
```

Comment:
Just curious: do you think anything would break if we don't include masking? On the one hand, not including masking means deviating from other engines, which could cause weird unexpected failures (what if some upgrade logic assumes masking is present? Maybe some support bundle collection logic makes this assumption too?), but on the other hand it's a little weird to include masking which we're not planning to use....

Reply:
I wasn't able to log in without masking, unfortunately.. probably a silly bug, since we don't usually have virt. without masking.. so, for now, I'd suggest we include masking to make forward progress, and look to remove it later if we want to.